app.js

```diff
@@ -1,6 +1,8 @@
 const path = require('path')
 const express = require('express')
 
+require('./server/leaderboard')

```

Comment:
Is require with side-effects good practice? Maybe add a function to call like `.start()`